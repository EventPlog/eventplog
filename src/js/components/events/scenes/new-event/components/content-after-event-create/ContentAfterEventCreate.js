import React from 'react'
import { Form, Label } from 'semantic-ui-react'
import { Link } from 'react-router-dom'
import styled from 'styled-components'

// local
import peopleDancing from '../../../../../../../img/giphys/black-guy-celebrating2.gif'
import { media } from '../../../../../../styles/mixins'
import Button from '../../../../../shared/button'

const StyledContent = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  
  > img {
    max-width: 500px;
    margin: 50px auto; 
    
    ${
      media.phone`
        width: 100%;
      `
    }
  }
  
  .medium.lowercase {
    text-transform: none;
    font-size: 1.2em;
    font-weight: 400;
    margin: 30px auto;
  }
  
  ul {
    font-size: 1.1em;
    font-weight: 400;
    line-height: 1.8em;
  }
`

const ContentAfterEventSubmit = ({ event, handleChange, loading, error }) => (
  <StyledContent className="text-center">
    <h3>You just created a event!!!</h3>
    <img src={peopleDancing} alt="people dancing" />
    <p>
      Thank you for all you do to strengthen the ecosytem.
    </p>
    <p>
      You event is not yet live since you might want to change the banner image, theme color, etc.
    </p>
    <p>
      To get the best out of eventplog as an event organizer, consider going through the following resources:
    </p>
    <ul>
      <li>
        <Link to="help.eventplog.com/working-with-untrained-volunteers">
          Setting up your event page.
        </Link>
      </li>
      <li>
        <Link to="help.eventplog.com/working-with-untrained-volunteers">
          Working best with untrained volunteers on event projects.
        </Link>
      </li>
      <li>
        <Link to="help.eventplog.com/working-with-untrained-volunteers">
          Handling check-ins in ways that sets you up for an engaging meetup.
        </Link>
      </li>
      <li>
        <Link to="help.eventplog.com/working-with-untrained-volunteers">
          Use feedback to improve your team at every opportunity.
        </Link>
      </li>
    </ul>
    <Button.Link to={`/communities/${event.community_id}/events/${event.id}`} inverted className="medium lowercase">
      Take me to my event so I can complete setup and publish.
    </Button.Link>
  </StyledContent>
)

export default ContentAfterEventSubmit

