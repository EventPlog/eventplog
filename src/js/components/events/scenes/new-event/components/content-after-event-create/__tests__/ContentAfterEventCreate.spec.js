import React from'react'
import ContentAfterEventCreate from '../ContentAfterEventCreate'
import {shallow, mount} from 'enzyme'

describe('ContentAfterEventCreate', () => {

  test('it renders correctly', () => {
    const wrapper = shallow(<ContentAfterEventCreate />)
    expect(wrapper).toMatchSnapshot()
  })
})