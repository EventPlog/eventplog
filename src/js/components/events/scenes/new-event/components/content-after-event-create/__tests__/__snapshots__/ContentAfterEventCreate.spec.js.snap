// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentAfterEventCreate it renders correctly 1`] = `
<styled.div
  className="text-center"
>
  <h3>
    You just created a event!!!
  </h3>
  <img
    alt="people dancing"
    src="black-guy-celebrating2.gif"
  />
  <p>
    Thank you for all you do to strengthen the ecosytem.
  </p>
  <p>
    You event is not yet live since you might want to change the banner image, theme color, etc.
  </p>
  <p>
    To get the best out of eventplog as an event organizer, consider going through the following resources:
  </p>
  <ul>
    <li>
      <Link
        replace={false}
        to="help.eventplog.com/working-with-untrained-volunteers"
      >
        Setting up your event page.
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="help.eventplog.com/working-with-untrained-volunteers"
      >
        Working best with untrained volunteers on event projects.
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="help.eventplog.com/working-with-untrained-volunteers"
      >
        Handling check-ins in ways that sets you up for an engaging meetup.
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="help.eventplog.com/working-with-untrained-volunteers"
      >
        Use feedback to improve your team at every opportunity.
      </Link>
    </li>
  </ul>
  <Component
    className="medium lowercase"
    inverted={true}
    to="/communities/undefined/events/undefined"
  >
    Take me to my event so I can complete setup and publish.
  </Component>
</styled.div>
`;
